src/routes/blog/spyder-6-remote-development/+page.md

```diff
@@ -0,0 +1,94 @@
+Therefore, we had to write an async API that was flexible enough to be able to call several async functions in a specific event loop, allow for loops to be run concurrently (to avoid blocking the main Qt event loop and causing Spyder's GUI to freeze), and be thread-safe so it can be called from any Qt thread.
+
+To cover all those requirements, Hendrick created the [`@AsyncDispatcher` decorator](https://github.com/spyder-ide/spyder/blob/v6.0.0/spyder/api/asyncdispatcher.py#L39) function, which starts a thread to run an async loop, schedules the async function in a specific loop, and returns a future object with the function's result.
+That result is emitted in a Qt signal which is then used by Spyder synchronously to perform other tasks (e.g. check if the server is running).
+
+
+![Spyder remote connection manager, showing a successfully connected host](remote-connection-manager-status.png)
+
+## What have you learned so far?
+
+Working on an open source project has provided me with the opportunity to witness how my contributions directly impact users in real time.
+Seeing my work being utilized and appreciated by the Spyder community is very rewarding and motivates me to strive for excellence and dedication.
+Additionally, observing individuals dedicated to improving Spyder for many years with a common goal to help the scientific community, and without focusing primarily on profit, is incredibly inspiring.
+
+This project has taught me the importance of adaptability and perseverance.
+There were moments with problems too hard to understand by myself, but with the team's support I was able to push through those obstacles, making me more confident and resilient.
+
+I'm very grateful for the opportunity to work on Spyder with such a dedicated and supportive team.
+I am excited to apply the lessons learned as we move into the next phases of this project!
+
+
+## What's next for the project?
+
+Our plans for the next feature version, [Spyder 6.1.0](https://github.com/spyder-ide/spyder/milestone/134), include adding the necessary APIs to `spyder-remote-services` so it can manipulate the remote file system, and grant Spyder the ability to work with, copy, move and delete remote files and directories right from the easy to use graphical interface.
+Users will also be able to create and manage local and remote Python environments, so they can work in different scientific and programming projects remotely at the same time.
+Finally, we plan to enable connecting to an already running JupyterHub instance, for situations where the user's school, enterprise or organization already has one set up that they want to work with using Spyder, instead of being locked in to the JupyterLab web interface.
+The changes required will be relatively straightforward since instead of talking to our own server for remote development, users will communicate with the existing JupyterHub instead.
```

Comment:
JupyterHub is the object here; at least in this context it just adds complexity to specifically call out communicating with a JupyterHub _server_ (which I _presume_ is what you intend here, but its potentially confusing) rather than just [the relevant part of] the existing JupyterHub which is the main point here.

Reply:
```suggestion
The changes required will be relatively straightforward since instead of talking to our own server for remote development, users will communicate with the existing JupyterHub instance instead.
```

Reply:
After @ccordoba12 explained this to me, agreed and modified this to

```suggestion
The changes required will be relatively straightforward since instead of talking to our own server for remote development, users will communicate with the existing JupyterHub instance instead.
```